src/main/java/uk/gov/hmcts/reform/translate/service/DictionaryService.java

```diff
@@ -50,4 +57,48 @@ public Map<String, String> getDictionaryContents() {
             throw new RoleMissingException(MANAGE_TRANSLATIONS_ROLE);
         }
     }
+
+
+    public void putDictionary(final Dictionary dictionaryRequest) {
+
+        val isManageTranslationRole = securityUtils.hasRole(MANAGE_TRANSLATIONS_ROLE);
+        val currentUserId = securityUtils.getUserInfo().getUid();
+        dictionaryRequest.getTranslations().entrySet()
+            .stream()
+            .forEach(englishPhrase -> processEachPhrase(englishPhrase, currentUserId, isManageTranslationRole));
```

Comment:
Just noticed that it is not the `englishPhrase` string being passed in, it is the map with both, so current name might be a bit misleading.
```suggestion
            .forEach(phrase -> processPhrase(phrase, currentUserId, isManageTranslationRole));
```

Reply:
DONE